tests/unittests/OnnxImporterTest.cpp

```diff
@@ -5520,3 +5520,143 @@ TEST_F(OnnxImporterTest, softmax13) {
                0.11920292, 0.880797, 0.880797, 0.11920292, 0.11920292, 0.880797,
                0.880797, 0.11920292, 0.11920292, 0.880797, 0.880797});
 }
+
+TEST_F(OnnxImporterTest, mixedPrecisionConfig) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  std::string netFilename(GLOW_DATA_PATH
+                          "tests/models/onnxModels/mixedPrecision.onnxtxt");
+  PlaceholderBindings bindings;
+  auto *F = mod.createFunction("main");
+  Placeholder *output;
+  Tensor data(ElemKind::FloatTy, {1, 3, 28, 28});
+
+  // Precision config info used in model loader
+  std::string precisionConfigFile(
+      GLOW_DATA_PATH "tests/models/onnxModels/mixedPrecision.yaml");
+  setModelLoaderPrecisionOpt(precisionConfigFile);
+
+  {
+    ONNXModelLoader onnxLD(netFilename, {"input"}, {&data.getType()}, *F);
+    output = EXIT_ON_ERR(onnxLD.getSingleOutput());
+    EXPECT_EQ(mod.getPlaceholders().size(), 2);
+    bindings.allocate(mod.getPlaceholders());
+  }
+  EE.compile(CompilationMode::Infer);
+  EE.run(bindings);
+  setConstantFoldLoaderOpsFlag(false);
+  auto result = bindings.get(output)->getHandle();
+  std::vector<dim_t> expectedDims = {1, 16, 26, 26};
+  EXPECT_TRUE(result.dims().vec() == expectedDims);
+
+  // This Model has three operators
+  // Here Conv and ReLu are specified in the yaml file to run in fp16.
+  //  Input: Placeholder(float)
+  //      |
+  //      V
+  //  Convolution(set to float16)
+  //      |
+  //      V
+  //  BatchNorm(float)
+  //      |
+  //      V
+  //  ReLu(set to float16)
+  //      |
+  //      V
+  //  Output: Placeholder(float)
+  // Check in the Glow graph the corresponding nodes are in FP16.
+  EXPECT_EQ(countNodeKind(F, Kinded::Kind::ConvertToNodeKind), 2);
+  for (auto &node : F->getNodes()) {
+    if (auto *MN = llvm::dyn_cast<MaxNode>(&node)) {
+      EXPECT_EQ(MN->getResult().getElementType(), ElemKind::Float16Ty);
+    } else if (auto *CN = llvm::dyn_cast<ConvolutionNode>(&node)) {
+      EXPECT_EQ(CN->getResult().getElementType(), ElemKind::Float16Ty);
+      auto *TN = llvm::dyn_cast<TransposeNode>(CN->getInput());
+      EXPECT_NE(nullptr, TN);
```

Comment:
nit: Since we use TN we need to assert, or else the test will segfault if the value is nullptr.
```suggestion
      ASSERT_NE(nullptr, TN);
```

Reply:
Done